### PR TITLE
Feature : Display template schema in related dialog

### PR DESCRIPTION
**Pull request by justinehell**
- Add carousel to display each schema in `NewProjectTemplateDialog`
- Display model schema in `ImportModelTemplateDialog`
- Handle zoom on click on each schema with `v-viewer` library

### CODE REVIEW

File: tests/unit/composables/TemplateManager.spec.js [hunk 1]
```diff
@@ -116,6 +125,24 @@ describe('Test composable: TemplateManager', () => {
...
+
+    it('should return filtered project type templates', async () => {
+      const componentTemplates = await TemplateManager.getTemplatesByType('project');
+
+      const result = new ComponentDefinition({
+        type: 'Project template',
+        icon: '/template-library/templates/project_template/icon.svg',
+      });
+
+      result.isTemplate = true;
```
Comment (k-y0u) [inline tests/unit/composables/TemplateManager.spec.js:137]: result = { ... };

File: src/components/dialog/ImportModelTemplateDialog.vue [hunk 2]
```diff
@@ -11,10 +11,20 @@
       {{ $t(`actions.models.import.dialog.name`) }}
     </template>
     <template v-slot:default>
-      <div class="q-pb-md text-subtitle2">
-        {{ $t('page.models.template.selectedTemplate', { template: modelTemplate?.type }) }}
+      <div class="q-pb-md text-center text-subtitle2">
+        {{ modelTemplate?.type }}
```
Comment (k-y0u) [inline src/components/dialog/ImportModelTemplateDialog.vue:15]: display "undefined" if modelTemplate does not exist ?

File: changelog.md [hunk 3]
```diff
@@ -67,5 +67,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Create ArrayInput
 - Add menu to see definition of component or attribute
 - Add model selection page + draw only one model in modelization page
+- Add carousel and image inside NewProjectTemplateDialog and ImportModelTemplateDialog respectively
```
Comment (k-y0u) [inline changelog.md:70]: carousel of images ?

**Review by k-y0u — CHANGES_REQUESTED** [review]

File: changelog.md (hunk 3 above)
Comment (justinehell) [reply changelog.md:70]: in one case its a carousel of images (NewProjectTemplateDialog) and in the other it's only one image (no carousel in ImportModelTemplateDialog)

File: tests/unit/composables/TemplateManager.spec.js (hunk 1 above)
Comment (justinehell) [reply tests/unit/composables/TemplateManager.spec.js:137]: nice ! done

File: src/components/dialog/ImportModelTemplateDialog.vue (hunk 2 above)
Comment (justinehell) [reply src/components/dialog/ImportModelTemplateDialog.vue:15]: I don't think `modelTemplate` can _not exist._ In any case, if it does not exist, it is not possible to open the dialog

File: changelog.md (hunk 3 above)
Comment (justinehell) [reply changelog.md:70]: changed

File: src/components/dialog/ImportModelTemplateDialog.vue (hunk 2 above)
Comment (justinehell) [reply src/components/dialog/ImportModelTemplateDialog.vue:15]: add `v-if="modelTemplate"` and remove optional chaining operator

**Review by Zorin95670 — APPROVED** [review]

**Pull request merged**